File: Web/godofgeeks23_AviralSrivastava_2024cse1056_2/gettingStarted/isanumberprime.cpp
#include <iostream>
using namespace std;
void isprime(int n)
{
    int divs = 2;
    for(int i=2;i<=n/2;i++)
        if(n%i==0)
            divs++;
    if(divs>2)
        cout<<"not prime"<<endl;
    else
        cout<<"prime"<<endl;
}
int main(int argc, char **argv){
    int t, n;
    cin >> t;
    while(t--)
    {
        cin>>n;
        isprime(n);
    }
    //write your code here
    
}

File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/GettingStarted/IsANumberPrime.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
  
       // write ur code here
        int t=scn.nextInt();
        for(int x=0;x<t;x++){
            int c=0;
            int n=scn.nextInt();
            for(int y=2;y*y<=n;y++){
                if(n%y==0){
                    c++;
                }
            }
            if(c==0){
                System.out.println("prime");
            }
            else{
                System.out.println("not prime");                
            }
        }
  }
  }